examples/cloud_run_cloudevents/README.md

```diff
@@ -0,0 +1,23 @@
+# Deploying a CloudEvent Function to Cloud Run with the Functions Framework
+
+This sample uses the [CloudEvents SDK](https://github.com/cloudevents/sdk-python) to send and receive a CloudEvent on Cloud Run.
+
+## How to run this locally
+
+Build the Docker image:
+
+```commandline
+docker build -t ff_example .
```

Comment:
I think we could have a better tag name here, maybe:

```suggestion
docker build -t cloudevent_example .
```

(I realize this is directly from the other sample).